test/plugins/auth.test.js

```diff
@@ -485,14 +486,34 @@ describe('auth plugin test', () => {
             })
         );
 
-        it('fails to issue a jwt given an invalid application auth token', () =>
+        it('returns user signed token given an API access token', () =>
             server.inject({
+                url: `/auth/token?access_key=${apiKey}`
```

Comment:
All of our access keys match `/[a-zA-Z0-9+/]{43}=/`, so if somebody picks an access key that matches that and it happens to be the same as one that we generate then there would be a collision.